README.md

```diff
@@ -2,11 +2,25 @@
 
 # algoliasearch-helper-js
 
-This module is the companion of the algoliasearch-client-js. It helps you keep
+This module is the companion of the [algoliasearch-client-js](https://github.com/algolia/algoliasearch-client-js). It helps you keep
 track of the search parameters and provides a higher level API.
 
-The helper is built on top of algoliasearch-client-js and this version is 
-specifically made to work with the newest V3 versions of it.
+This is the plugin you will need to easily build a good search UX.
```

Comment:
Good thing that you add an explanation here. On the vocabulary side, I would say that this is more of a wrapper than a plugin. Do you think that plugin will be more easily understood?